test/latin-var-name-only.test.js

```diff
@@ -1,10 +1,10 @@
-const latinVarNameOnly = require('../src/rules/latin-var-name-only');
```

Comment:
This path should stay the same. You should use `/src/rules/latin-var-name-only/index.js` file to manage internals. Think about how this rule will become a part of the plugin and not only tests